src/delimiter.js

```diff
@@ -317,6 +312,71 @@ const makeStackedDelim = function(delim, heightTotal, center, options, mode,
         Style.TEXT, options, classes);
 };
 
+const sqrtSpan = function(height, delimData, options) {
+    // Create a span containing an SVG image of a sqrt symbol.
+    // Each viniculum will stretch to match the inner group's width.
+    let span;
+
+    if (delimData.type === "small") {
+        // Get an SVG that is derived from glyph U+221A in font KaTeX-Main.
+        span = buildCommon.makeSpan(["stretchy", "sqrt-main"], [], options);
+        span.height = 1;  // Inside the font, the glyph is 1000 units tall.
+
+        // span.depth is really zero, but I'll mis-use the .depth attribute
+        // to carry info about surd width back to groupTypes.sqrt
+        span.depth = 0.781;   // surd width, from the font.
```

Comment:
Good idea, to use `span.width`. I'll do that.

Re: multiplier. (1) Because it works empirically.  (2) The function `traverseSequence` has already taken care of that for small delimiters. The relevant code being:
````
        // Small delimiters are scaled down versions of the same font, so we
        // account for the style change size.

        if (sequence[i].type === "small") {
            const newOptions = options.havingBaseStyle(sequence[i].style);
            heightDepth *= newOptions.sizeMultiplier;
        }
````